handsontable/src/plugins/filters/ui/multipleSelect.js

```diff
@@ -185,9 +186,29 @@ class MultipleSelectUI extends BaseUI {
         fillHandle: false,
         fragmentSelection: 'cell',
         tabMoves: { row: 1, col: 0 },
-        beforeKeyDown: event => this.onItemsBoxBeforeKeyDown(event)
       });
       this.itemsBox.init();
+
+      const shortcutManager = this.itemsBox.getShortcutManager();
+      const gridContext = shortcutManager.getContext('grid');
+      const config = { group: SHORTCUTS_GROUP };
+
+      gridContext.addShortcuts([{
+        // TODO: Are those shortcuts really needed?
+        keys: [['ArrowUp'], ['ArrowDown'], ['ArrowLeft'], ['ArrowRight'], ['Tab'], [' '], ['Enter']],
```

Comment:
Can I propose to implement the `'Space'` key name in favor of `' '`?  For example: `[['ArrowUp'], ['ArrowDown'], ['ArrowLeft'], ['ArrowRight'], ['Tab'], ['Space'], ['Enter']],`. With an empty string and a whitespace within it it looks weird 🤔  There are other places in the code that I would change.

Reply:
I'm afraid that it won't be so easy without mapping. We are using `event`'s `key` property. There are only two [KeyboardEvent](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent)'s properties which aren't deprecated. That are `code` and `key` elements.

I spoke about that with @warpech some times ago. Using `key` isn't perfect solution. Maybe `alt` + another key shortcut isn't often used (in our code not once), but for example `alt` + `s` shortcut should be defined as `ś` on polish keyboard. Alternative of using `code` isn't the best too. It's highly related to physically pressed keys. So, not only for such weird situation as described above, but for almost every shortcut there should be considered type of keyboard used (whether it's QWERTY, QWERTZ and AZERTY keyboard, and so on).

Reply:
Correct me if I'm wrong but supporting mapping shouldn't be so hard to implement. The [`normalizeKeys`](https://github.com/handsontable/handsontable/blob/feature/issue-8942/handsontable/src/shortcuts/context.js#L71) could implement that feature. So while adding a shortcut with the `Space` value underneath translates to an empty string.

Reply:
Of course that mapping isn't hard to implement. As i wrote in first sentence I don't consider mapping at first. I don't know whether it's proper behaviour. Which keys should be mapped - just space? How to document it?

I showed some architecture decision to show broader context and asked whether it's correct.

Reply:
I think that `Space` would be enough at this moment. This is the only key that can be used broadly by other developers. It can be misunderstood and can lead to typos such as`' '` and `''`.

Reply:
Using the `key` property might not be ideal (it proved to be potentially problematic in #9230), but I'd go with it + mapping. At first, it might be just for `Space`, but if the infrastructure's there, we can always add some other aliases if needed.

Reply:
Yeah, I agree with mapping just `Space`. The biggest problem with `' '` is that it is almost unsearchable.

@wszymanski thanks for your explanations in https://github.com/handsontable/handsontable/commit/f195015d4d284e0ea9070d7aee20ecee5e5629ad and https://github.com/handsontable/handsontable/commit/cc6a1ba95bd35bbbc1530b1654e2bb6bbd623e59, they are very welcome.

Reply:
Thank you, after discussion on Slack, there have been found also another elements needing mapping:

`[1] Internet Explorer, Edge (16 and earlier), and Firefox (36 and earlier) use "Left", "Right", "Up", and "Down" instead of "ArrowLeft", "ArrowRight", "ArrowUp", and "ArrowDown".`

`[1] Older browsers may return "Spacebar" instead of " " for the Space Bar key. Firefox did so until version 37, as did Internet Explorer 9, 10, and 11.`

Reply:
It was changed within 67e159d57608d9d589d48f0de68a01093dcb5189 and ab95a678c05af3ddc8f9fa47be3468763d98300d.